src/puptoo/process/profile.py

```diff
@@ -665,6 +664,13 @@ def system_profile(
                         "image": bootc_value.get('image', {}).get('image', {}).get('image', ''),
                         "image_digest": bootc_value.get('image', {}).get('imageDigest', ''),
                     }
+            cached_value = (status.get("staged", {}).get("cachedUpdate", {}) or
```

Comment:
nitpick:
```
status.get("staged", status.get("booted", {}).get("cachedUpdate", {})
```

Reply:
@xiangce, just this way won't give us the expected result. For example. `status.staged` has data but without `status.staged.cachedUpdate` case ( like `BOOTC_STATUS_WITH_CACHEDUPDATE_2` ), this will always skip the `status.booted` part.